electron/main.ts

```diff
@@ -374,65 +378,97 @@ ipcMain.on('callTool', async (event, { tool, wine, prefix, exe }: Tools) => {
     command = `WINEPREFIX='${winePrefix}' ${wineBin} '${exe}'`
   }
 
-  console.log({ command })
+  Logger.info({message: `Trying to run command ${command}`, service: 'ipcMain::callTool::executingCommand'})
   return await execAsync(command)
 })
 
 /// IPC handlers begin here.
 
-ipcMain.handle('checkGameUpdates', () => LegendaryLibrary.get().listUpdateableGames())
+ipcMain.handle('checkGameUpdates', async () => {
+  await Logger.info({message: 'Checking all apps for updates', service: 'ipcMain::checkGameUpdates'})
+  return LegendaryLibrary.get().listUpdateableGames()
+})
 
 // Not ready to be used safely yet.
-ipcMain.handle('updateAll', () => LegendaryLibrary.get().updateAllGames())
+ipcMain.handle('updateAll', async () => {
+  await Logger.info({message: 'Updating all apps', service:'Game::updateAllGames'})
+  return LegendaryLibrary.get().updateAllGames()
+})
 
-ipcMain.handle('checkVersion', () => checkForUpdates())
+ipcMain.handle('checkVersion', async () => {
+  await Logger.info({message: 'Checking for a newer heroic version', service: 'ipcMain::checkVersion'})
```

Comment:
I'm worried that all those await calls will make the app slower. Because we always need to write something to the logs first to then do something.
Would be good to benchmark those.
Especially on an HDD that is slower.

Reply:
As the writes are very small, I doubt it actually makes a noticeable difference. However, I'm now looking into doing some sort of worker process for the logger instead.